src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work");
+    }
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped its work");
+    }
+}
```

Comment:
add an empty line) the same for other classes

Reply:
empty lines at the end were added.
Class Printer was removed.
mvn clean package result is NO errors.